book/chapters/pitfalls.tex

```diff
@@ -137,10 +137,12 @@ \section{Pitfall: Missing glyphs}
 The Unicode Standard is often praised (and deservedly so) for solving many of
 the perennial problems with the interchange and display of the world's writing
 systems. Nevertheless, a common complaint from users is that some symbols do not display 
-correctly, i.e.\ \textit{not at all} or from a \textit{fall back font}, e.g.\ a 
+correctly, i.e. they might be displayed \textit{not at all} or only from a so-called 
+\textit{fall back font}, e.g.\ showing a 
 rectangle <▯>, question mark <?>, or the Unicode replacement character <�>. 
-The user's computer does not have the fonts 
-installed that map the desired glyphs to Unicode characters. Therefore the glyphs cannot be displayed.
+The reason for such behaviour is that the user's computer does not have the fonts 
```

Comment:
@cysouw - no British English spelling ("behaviour") -- it took me a long time to get the book consistently American English spelling :)